final_task/setup.py

```diff
@@ -0,0 +1,18 @@
+import sys
+sys.path.append('rss_reader')
+from setuptools import setup, find_packages
+from rss_reader import rss_reader
```

Comment:
This will not work on clean machine, if there are no libs from `requirements.txt` installed.

Reply:
I tried running on a clean machine.  When libraries is installed, just start it as rss_reader.rss_reader.
Example:
python -m rss_reader.rss_reader "http://news.yahoo.com/rss/" --limit 2 --verbose

Reply:
> When libraries is installed


Yes, _when_ libraries are installed. The idea of second iteration is that your app will download required libs itself, without you doing `pip install -r requirements.txt`

> python -m rss_reader.rss_reader "http://news.yahoo.com/rss/" --limit 2 --verbose

Your app works perfectly while using `python rss_reader...`, But the idea of second iteration is to install a CLI utility, that will allow you to run your app like this:
```
rss-reader <rss_source> <options>
```
Without manually installing required libs.